public/views/signUpView.js

```diff
@@ -38,19 +38,18 @@ export default class SignUpView {
         this._passwordErrorField = document.getElementById('js-password-error');
         this._passwordRepeatField = document.getElementById('js-repeat-password-input');
         this._passwordRepeatErrorField = document.getElementById('js-repeat-password-error');
-        this._error = document.getElementById('js-sign-up-error');
-
-        this._regBtn = document.getElementById('js-sign-up-btn')
-        this._logBtn = document.getElementById('js-have-account-btn')
-
-        this._error.textContent = userStore.user.errorReg;
 
+        this._regBtn = document.getElementById('js-sign-up-btn');
+        this._logBtn = document.getElementById('js-have-account-btn');
     }
 
     _addPagesListener() {
         this._regBtn.addEventListener('click', () => {
             if (this._validateFirstName && this._validateLastName && this._validateEmail && this._validatePassword && this._validatePasswordRepeat) {
                 actionUser.signUp({firstName: this._firstNameField.value, lastName: this._lastNameField.value, email: this._emailField.value, password: this._passwordField.value});
+            } else {
+                userStore.user.errorReg = 'Необходимо заполнить корректно все поля';
```

Comment:
Мне кажется, тут лучше без "необходимо", да и не очень ясно, что такое "корректно"

```suggestion
                userStore.user.errorReg = 'Заполните все указанные поля';
```

Если хочется корректно, то лучше

```suggestion
                userStore.user.errorReg = 'Заполните корректно все поля';
```

![изображение](https://user-images.githubusercontent.com/31134795/233519896-7ff9f6d5-478b-4390-a640-73a85d46066e.png)